### DISCUSSION TITLE
Consulta ComunicacaoCTe e ComunicacaoNFe existe?

### DISCUSSION THREAD

**Discussion by osielgomes** [TadaSoftware/PyNFe#284] [General]
Boa tarde!!!

Gostaria de saber se teria como realizar a seguinte situação: 
- Determinar uma consulta de DFe somente para CT-e  e outra somente para NF-e. 
Pude observar que se eu executar desta forma:

**`con_cte = ComunicacaoCTe(uf, certificado, senha, homologacao)`**
 **`xml_cte = con_cte.consulta_distribuicao(cnpj=CNPJ, chave=CHAVE, nsu=NSU)`**

Ele me traz uma consulta somente de Ct-e, existe a mesma opção somente para NF-e, uma do tipo ComunicacaoNFe? Se sim como usar?

Obrigado!!!

**Comment by leogregianin** [comment]
Bom dia @osielgomes, existe a mesma funcionalidade porém com o nome do método diferente, veja aqui: https://github.com/TadaSoftware/PyNFe/wiki/Fluxo-da-Consulta-de-Distribuicao-por-NSU

Dá uma olhada no wiki https://github.com/TadaSoftware/PyNFe/wiki com todos os exemplos da lib.
